src/view/reviewManager.ts

```diff
@@ -667,7 +668,9 @@ export class ReviewManager {
 			this._createPullRequestHelper = new CreatePullRequestHelper(this.repository);
 			this._createPullRequestHelper.onDidCreate(async createdPR => {
 				await this.updateState();
-				await openDescription(this._context, this._telemetry, createdPR, this._folderRepoManager, this);
+				const treeNodes = await this.changesInPrDataProvider.getChildren();
+				const descriptionNode = treeNodes[0] as DescriptionNode;
```

Comment:
To avoid relying on position 0 and casting, you could expose a `getDescriptionNode` on `PullRequestChangesTreeDataProvider`.

Reply:
Thanks, adding this!